run-e5.py

```diff
@@ -0,0 +1,90 @@
+import asyncio
+
+import numpy as np
+from typing import Any
+from openai import AsyncOpenAI
```

Comment:
You should move imports of external library inside the class to make it optional